io/private/las/Vlr.cpp

```diff
@@ -320,6 +320,13 @@ std::istream& operator>>(std::istream& in, las::Evlr& v)
 
             v.dataFunc = MetadataFunc(metadataId);
         }
+        else if (el.key() == "evlr")
+        {
+            if (!el.value().is_boolean())
+                throw pdal_error("LAS VLR metadata key must be specified as a string.");
```

Comment:
You've required it to be a boolean, not a string, right?

Reply:
updated as suggested in 4a59f9dcbfcecec7dfb97f64adec49ac1c783c36